src/Command/AssertBackwardsCompatible.php

```diff
@@ -69,7 +70,7 @@ protected function configure(): void
                 'format',
                 null,
                 InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY,
-                'Currently supports "console", "markdown" or "github-actions"',
+                'Currently supports "console", "markdown", "github-actions" or "junit"',
```

Comment:
I wonder if it's possible to give the options as a `list<string>` to `addOption()`, so it prints the possible values (and does the validation) later?

Reply:
the 6th argument to `addOption` `$suggestedValues` appears to take an array of values, but when used as such:

```php
            ->addOption(
                'format',
                null,
                InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY,
                'The output format to use.',
                ['console'],
                ['console', 'markdown', 'github-actions', 'junit'],
            )
```

the output looks like this: 

```
      --format[=FORMAT]                   The output format to use. [default: ["console"]] (multiple values allowed)
```

which unfortunately does not list the values

it appears as though the `$suggestedValues` parameter powers the autocompletion script made available in symfony console, and is not used to inform / validate the input beyond that

Reply:
i will still add the 6th argument to enable the autocompletion however